pkg/pubsub/operations/subscription.go

```diff
@@ -128,26 +172,49 @@ func (s *SubscriptionOps) Run(ctx context.Context) error {
 		logger.Info("Previously created.")
 
 	case ActionCreate:
+		// Load the topic.
+		topic, err := s.getTopic(ctx)
+		if err != nil {
+			logger.Fatal("Failed to get topic.", zap.Error(err))
```

Comment:
Seems really bad to `logger.Fatal` in a library.

Reply:
This kills the job and marks it failed. working as intended.

Reply:
ok I will fix.

Reply:
Fixed.